com.rustdesk.RustDesk.metainfo.xml

```diff
@@ -28,16 +30,31 @@
   </categories>
   <screenshots>
     <screenshot type="default">
+      <caption>Remote desktop session</caption>
       <image>https://user-images.githubusercontent.com/71636191/171661982-430285f0-2e12-4b1d-9957-4a58e375304d.png</image>
     </screenshot>
   </screenshots>
+  <branding>
+    <color type="primary" scheme_preference="light">#00bbe3</color>
+    <color type="primary" scheme_preference="dark">#0176fa</color>
```

Comment:
Can you please explain where you see these colors on the Rustdesk website/repository?

Reply:
I tried looking for a brand guide for RustDesk to fetch colors from, but didn't find anything. Ended up with extracting colors from the app icon (brightest/darkest part of the gradient) and adjusting them for contrast with black/white text respectively, since they could e.g. end up being by various app store frontends to generate banners:

![Skjermbilde fra 2024-10-09 13-06-29](https://github.com/user-attachments/assets/0250fe77-f3d1-4569-b434-06e8ad531f21)

If you have any better suggestions / specific colors used by RustDesk for the same purpose in other places it would of course make sense to go with that instead though.

Reply:
@rustdesk please provide your feedback on these colors. Do they meet your expectations? As you can see, these colors will be used on the Flathub website in their promotional banners.

Reply:
Looks ugly, :(

Reply:
I would suggest something like below.
![image](https://github.com/user-attachments/assets/57a88087-960d-4efb-b3f4-77725ac817b7)
![image](https://github.com/user-attachments/assets/8ce14e72-2e7d-4120-b8f0-20f60356bf37)

Reply:
While the dark blue could work as the dark brand color, the light one would ideally be something other than white given that it’ll be displayed on top of a white background.

Reply:
Here's an attempt at using more neutral colors:

![Skjermbilde fra 2024-10-10 16-51-08](https://github.com/user-attachments/assets/fba23f79-110e-4788-8827-a95abab3fbaa)

Does this direction work?

Reply:
How about #040620 for dark?

Reply:
I'm concerned about the contrast with the dark background, but we can move forward with that if you think it's fine:

![Skjermbilde fra 2024-10-10 17-15-32](https://github.com/user-attachments/assets/e891a431-09d5-42d2-a394-9ab5c910be23)

Reply:
How about #0160EE?

Reply:
Works perfectly.